apps/admin-x-activitypub/src/hooks/use-activity-pub-queries.ts

```diff
@@ -955,6 +927,29 @@ export function useAccountFollowsForUser(handle: string, type: AccountFollowsTyp
     });
 }
 
+function updateActivityInCache(queryClient: QueryClient, queryKey: unknown[], id: string, updated: Partial<Activity>) {
```

Comment:
I recently added a method to update an activity, can that be used instead of this or does this do something different?